### PR TITLE
Feature/schedule page

### PR DESCRIPTION
**Pull request by trizoza**
Adds SCHEDULE page with upcoming 5 days of broadcast.

https://deploy-preview-45--ehfm.netlify.app/

### CODE REVIEW

File: src/containers/residents/Residents.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const ResidentsContainer = ({ residentsData }) => {
       <MetaData
         title="Residents | EHFM"
         description="Get to know our presenters and listen back to the archive of previous shows."
-        ult="http://www.ehfm.live/residents"
```
Comment (RichardPK) [inline src/containers/residents/Residents.js:18]: 🤦

File: src/hooks/useScheduleData.js [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 export const useScheduleData = () => {
   const [airTimeSchedule, setAirtimeSchedule] = useState(null);
+  const [fiveDayAirTimeSchedule, setFiveDayAirtimeSchedule] = useState(null);
   const [scheduleData, setScheduleData] = useState(null);
 
   const scheduleApiCall = useCallback(() => {
     fetch("https://ehfm.airtime.pro/api/week-info")
       .then((response) => response.json())
       .then((data) => {
+        delete data.AIRTIME_API_VERSION;
```
Comment (RichardPK) [inline src/hooks/useScheduleData.js:13]: What's this about?

File: src/containers/shedule/Schedule.jsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        mixcloudWidgetHtml={mixcloudWidgetHtml}
+        cookiesBannerShowing={!cookies.ehfm}
+      >
+        <ScheduleData>
+          {fiveDayAirTimeSchedule && residentsData
+            ? Object.keys(fiveDayAirTimeSchedule).map((day, i) => {
+                return (
+                  <>
+                    <Name key={`day-${day}-${i}`}>
+                      {fiveDayAirTimeSchedule[day].date.split("NEXT").pop()}
```
Comment (RichardPK) [inline src/containers/shedule/Schedule.jsx]: Might help readability by assigning `fiveDayAirTimeSchedule[day]` to a variable?

File: src/components/nav-bar/socials/Socials.js [hunk 4]
```diff
@@ -19,7 +19,7 @@ const Socials = () => {
         </a>
       </SocialInnerWrapper>
       {/* <SocialInnerWrapper>
-        <a href="http://eh-fm.chatango.com/" target="blank">
+        <a href="https://eh-fm.chatango.com/" target="blank">
           <NavChat src={ChatangoLogo} alt="chatango" />
         </a>
       </SocialInnerWrapper> */}
```
Comment (RichardPK) [inline src/components/nav-bar/socials/Socials.js]: Can probably just be killed?

File: src/App.js [hunk 5]
```diff
@@ -33,6 +33,10 @@ export const App = () => {
       return Boolean(essentialForAllPaths && supportPageData);
     }
 
+    if (path === "/schedule") {
+      return true;
+    }
```
Comment (RichardPK) [inline src/App.js:38]: Not really a fan of my 'essentialDataFetchingFinished' function at all. Typescript + ReactQuery would help improve this loading behaviour massively, one for another time.

**Review by RichardPK — APPROVED** [review]

File: src/hooks/useScheduleData.js (hunk 2 above)
Comment (trizoza) [reply src/hooks/useScheduleData.js:13]: The response returns API version, but later I'm looping through the `Object.keys` and I'm only interested in the days keys. 

Screenshot 2022-11-13 at 20 57 03

File: src/containers/shedule/Schedule.jsx (hunk 3 above)
Comment (trizoza) [reply src/containers/shedule/Schedule.jsx]: Good spot - [fixed](https://github.com/RichardPK/eh-fm/pull/45/commits/6f074cd5ee5ec1d2b1672b91f75f9714bb70f583)

File: src/components/nav-bar/socials/Socials.js (hunk 4 above)
Comment (trizoza) [reply src/components/nav-bar/socials/Socials.js]: [Removed](https://github.com/RichardPK/eh-fm/pull/45/commits/6f074cd5ee5ec1d2b1672b91f75f9714bb70f583)

File: src/App.js (hunk 5 above)
Comment (trizoza) [reply src/App.js:38]: When you were writing this, `useQuery` was probably only just an idea in Tanner's head ;)

**Pull request merged**